jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
     preset: 'ts-jest',
-    testEnvironment: 'node',
+    testEnvironment: 'jsdom',
```

Comment:
Is this ok? I needed this for the viewportstore test as it uses `window.matchMedia`.

Reply:
I have no preference here. I think I just put in, what the "getting-started-with-jest" tutorial says.

Reply:
I reverted. jsdom sounds more correct but who knows and as long as it works with the current value no need to change